test.py

```diff
@@ -0,0 +1,166 @@
+from sme_finance_application_schema.translations import *
+
+
+# The following are fields that do not appear in the objects when they are translated from finance_application_v3
+UNTRANSLATED_SME_V5_FIELDS = [
+    'date_of_first_filed_accounts', #To remove - no translation written
+]
+UNTRANSLATED_CONTACT_V3_FIELDS = [
+    'county', #Deprecated
+]
+
+
+# The following are fields that do not appear in the objects when they are translated from sme_v5 / sme_contact_v3
+UNTRANSLATED_ENTITY_V1_FIELDS = [
+    'employees', #Not present in SME_v5
+    'registration_date', #Not present in SME_v5
+    'addresses', #Not present in SME_v5
+    'date_of_first_filed_accounts', #To remove - no translation written
+    'free_form', #Not present in SME_v5
+]
+UNTRANSLATED_FINANCE_APPLICATION_V3_FIELDS = [
+    'actors', #Unsupported
+]
+UNTRANSLATED_PERSON_V1_FIELDS = [
+    'date_of_birth' #Not present in SME_V5 or CONTACT_V3
+]
+UNTRANSLATED_FINANCE_NEED_V1_FIELDS = [
+    'free_form', #Not present in SME_v5
+    'property_ownership', #Not present in SME_v5
+    'permission_for_development', #Not present in SME_v5
+    'property_development_type', #Not present in SME_v5
+    'property_work_started', #Not present in SME_v5
+    'asset_type', #Not present in SME_v5
+    'type_of_mortgage', #Not present in SME_v5
+    'experience_in_development', #Not present in SME_v5
+    'deposit', #Not present in SME_v5
+    'vehicle_type', #Not present in SME_v5
+    'type_of_property', #Not present in SME_v5
+]
+UNTRANSLATED_ADDRESS_V1_FIELDS = [
+    'locality_name', #TODO
+]
+
+
+def patch_store(store):
+    for schema in ('entity_v1', 'person_v1', 'finance_need_v1', 'address_v1', 'actor_v1'):
+        resource = resource_filename('sme_finance_application_schema', schema)
+        with open(resource) as f:
+            store['https://www.fundingoptions.com/schema/' + schema] = json.loads(f.read())
+
+
+class TestJson(TestCase):
+    def test_entity_json(self):
+        for schema in ('entity_v1', 'person_v1', 'finance_need_v1', 'address_v1', 'finance_application_v1', 'finance_application_v2', 'finance_application_v3', 'batch_application_v1', 'batch_response_v1', 'actor_v1'):
+            with self.subTest(schema=schema):
+                resource = resource_filename('sme_finance_application_schema', schema)
+                with open(resource) as f:
+                    content = f.read()
+                self.assertTrue(json.loads(content))
+
+
+def test_validity_of_data(data, schema_name):
+    resource = resource_filename('sme_finance_application_schema', schema_name)
+    with open(resource) as f:
+        schema = json.loads(f.read())
+        validator = jsonschema.validators.Draft4Validator(schema)
+        patch_store(validator.resolver.store)
+        if not validator.is_valid(data):
+            raise Exception('invalid data according to {}: {}'.format(schema_name, validator.validate(SME_V5)))
+
+
+class TestTranslations(TestCase):
```

Comment:
I don't know that the first two test_ methods belong in this class - they are not translations. *If* you were minded to clean it up I would suggest splitting them out + using a list of (data, schema) tuples and unittest subTests (c.f. `test_mapping_trading_history_to_reduced_option_set` in `application/tests/tests.py`).